components/automate-ui/src/app/components/chef-components.module.ts

```diff
@@ -12,6 +12,7 @@ import { AuthorizedComponent } from './authorized/authorized.component';
 import { BreadcrumbsComponent } from './breadcrumbs/breadcrumbs.component';
 import { BreadcrumbComponent } from './breadcrumb/breadcrumb.component';
 import { CalendarComponent } from './calendar/calendar.component';
+import { CreateV1TeamModalComponent } from '../page-components/create-v1-team-modal/create-v1-team-modal.component';
```

Comment:
It seems a bit strange to me that CreateV1TeamModalComponent would need to live here when all of the page-components are in appModule, however I noticed that the other Modals that are being used in Team Management pages are all in ChefComponentsModule.  

I don't know enough about how module files work yet to understand why `team-management.component` doesn't have access to CreateV1TeamComponent when its part of appModule.